parquet_file/src/catalog.rs

```diff
@@ -110,11 +115,17 @@ pub enum Error {
     #[snafu(display("Upgrade path not implemented/supported: {}", format))]
     UnsupportedUpgrade { format: String },
 
-    #[snafu(display("Parquet already exists in catalog: {}", path))]
-    ParquetFileAlreadyExists { path: String },
+    #[snafu(display("Parquet already exists in catalog: {:?}", path))]
+    ParquetFileAlreadyExists { path: DirsAndFileName },
+
+    #[snafu(display("Parquet already does not exists in catalog: {:?}", path))]
+    ParquetFileDoesNotExists { path: DirsAndFileName },
```

Comment:
And to keep the type name matching the message, this should probably be:

```suggestion
    ParquetFileDoesNotExist { path: DirsAndFileName },
```